### ISSUE TITLE
I'm using the v4 model to fine-tune in Korean, but I think the loss is strange.

### ISSUE THREAD

**Issue by yongmin96** [RVC-Boss/GPT-SoVITS#2595]
<img width="1137" height="556" alt="Image" src="https://github.com/user-attachments/assets/e7ce1177-e88b-40f6-9e6f-eff44e2e8860" />

Hello!. I'm fine-tuning the model for another language using 400 hours of data, with a batch size of 3 and a learning rate of 1e-5. However, the losses after 3 epochs seem odd. Any recommendations or insights on what might be going wrong?

**Comment by bahodir4** [comment]
@yongmin96 hello, could you help me to guide how to fine-tune the model with a new language, I saw they have trained some specific languages, but I wanted to train mine, so it'd be great if you share your guidance, Thanks!!!

**Comment by jmdBB** [comment]
Hey team, I'm also interested in how to train the model for my own languages, european ones mostly... I'm willing to help somehow also, appreciate!
